sorc/fv3gfs_build.cfg

```diff
@@ -9,14 +9,15 @@
  Building ufs_utils (ufs_utils) ........................ yes
  Building gldas (gldas) ................................ yes
  Building gfs_wafs (gfs_wafs) .......................... yes
- Building gaussian_sfcanl (gaussian_sfcanl)............. yes
- Building enkf_chgres_recenter (enkf_chgres_recenter) .. yes
- Building enkf_chgres_recenter_nc (enkf_chgres_recenter_nc) .. yes
- Building tropcy_NEMS (tropcy) ......................... yes
- Building gfs_fbwndgfs (gfs_fbwndgfs) .................. yes
- Building gfs_bufrsnd (gfs_bufrsnd) .................... yes
- Building fv3nc2nemsio (fv3nc2nemsio) .................. yes
- Building regrid_nemsio (regrid_nemsio) ................ yes
+ Building gaussian_sfcanl (gaussian_sfcanl)............. no
+ Building enkf_chgres_recenter (enkf_chgres_recenter) .. no
+ Building enkf_chgres_recenter_nc (enkf_chgres_recenter_nc) .. no
+ Building tropcy_NEMS (tropcy) ......................... no
+ Building gfs_fbwndgfs (gfs_fbwndgfs) .................. no
+ Building gfs_bufrsnd (gfs_bufrsnd) .................... no
+ Building fv3nc2nemsio (fv3nc2nemsio) .................. no
+ Building regrid_nemsio (regrid_nemsio) ................ no
```

Comment:
why no? we are building them as part of `workflow_utils`.

Reply:
never mind. 
I suppose this will be cleaned up later.

Reply:
The ones set to "no" will be removed during the cleanup/decommissioning stage.